com.mojang.Minecraft.appdata.xml

```diff
@@ -51,6 +51,7 @@
   <project_license>LicenseRef-proprietary</project_license>
   <developer_name>Mojang AB</developer_name>
   <releases>
+    <release version="2.1.12872" date="2020-03-16"/>
```

Comment:
@barthalion Is there any way to disable AppData updates by the extra data checker? We have the game releases in AppData, not the client ones.

Reply:
Doesn't the launcher download whatever is considered the latest Minecraft release?

Reply:
Yeah, it does, and because of this, I don't see how useful is the launcher version information to users. We always had versions of the Minecraft game in AppData, not the launcher ones.

Reply:
Honestly I don't see anything useful in version of game that is not managed by Flatpak and Flathub. The launcher is the software being shipped.

Reply:
Ok, fair point. I will remove the game versions from AppData and merge this PR.

Reply:
To be clear, it's your decision, I'm just sharing what in my opinion makes more sense. Currently external-data-checker does not allow to disable appdata updates.